styles/mixin.scss

```
@@ -0,0 +1,23 @@
+@mixin desktop {
+  @media(max-width:1920px){
+    @content;
+  }
+}
+
+@mixin large-tablet{
+  @media(max-width:1279px){
+    @content;
+  }
+}
+
+@mixin tablet{
+  @media(max-width:1023px){
+    @content;
+  }
+}
+
+@mixin mobile{
+  @media(max-width:767px){
+      @content;
+  }
+}
```

Make easier to read.
Can write the variables to global variables.scss
The logic of `desktop` here looks like a `laptop` logic. 


```suggestion
$width-desktop: 1960px;
$width-laptop: 1440px;
$width-large-tablet: 1024px;
$width-tablet: 768px;

@mixin laptop{
  @media(max-width: ($width-desktop - 1px)){
    @content;
  }
}

@mixin large-tablet{
  @media(max-width: ($width-laptop - 1px)){
    @content;
  }
}

@mixin tablet{
  @media(max-width: ($width-large-tablet - 1px)){
    @content;
  }
}

@mixin mobile{
  @media(max-width: ($width-tablet - 1px)){
      @content;
  }
}

```